README.md

```diff
@@ -1,22 +1,628 @@
+
+<br/>
+<br/>
+
+#### Navigating Argo UI
+
+The 'Workflows' tab on the left side menu shows you all running workflows. If you click a current workflow, it will show you a schematic of the jobs spread across multiple instances. 
+
+<img width="1000" alt="Screenshot 2025-06-20 at 12 48 46 PM" src="https://github.com/user-attachments/assets/bd6bd991-f108-4be9-a1aa-6cb0f1ab1db5" />
+
+<br/>
+<br/>
+
+If you click on a specific job, it will show you lots of information of the process including which VM it is running on, the duration of the process, and logs. 
+
+<img width="1000" alt="Screenshot 2025-06-20 at 12 58 55 PM" src="https://github.com/user-attachments/assets/ab10f2b4-3120-47be-b1dd-601687707f0c" />
+
+<br/>
+<br/>
+
+A successfull argo run 
+
+<img width="650" height="789" alt="argo_success" src="https://github.com/user-attachments/assets/201b0594-7557-4d85-a99b-677e6c173a44" />
+
+<br/>
+<br/>
+
+
+### 4. Metashape Outputs
+The metashape outputs will be written to `/ofo-share/argo-outputs/<RUN_FOLDER>`. Each dataset will have its own subdirectory in the <RUN_FOLDER>. Output imagery products (DEMs, orthomosaics, point clouds, report) will be written to `/ofo-share/argo-outputs/<RUN_FOLDER>/<dataset_name>/output`. Metashape projects .psx will be written to `/ofo-share/argo-outputs/<RUN_FOLDER>/<dataset_name>/project`.
+
+```bash
+/ofo-share/
+├── argo-input/
+│   ├── config.yml
+│   ├── datasets.txt
+│   ├── benchmarking-greasewood/
+│   │   ├── image_01.jpg
+│   │   └── image_02.jpg
+│   └── benchmarking-swetnam-house/
+│       ├── image_01.jpg
+│       └── image_02.jpg
+└── argo-output/
+    └── <RUN_FOLDER>/
+        ├── benchmarking-greasewood/
+        │   ├── output/
+        │   │   ├── orthomosaic.tif
+        │   │   ├── dsm.tif
+        │   │   └── point-cloud.laz
+        │   └── project/
+        │       └── metashape_project.psx
+        └── benchmarking-swetnam-house/
+            ├── output/
+            │   ├── orthomosaic.tif
+            │   ├── dsm.tif
+            │   └── point-cloud.laz
+            └── project/
+                └── metashape_project.psx
+```
+
+
+
+<br/>
+<br/>
+<br/>
+<br/>
+
+### 5. Argo Workflow Logging in postGIS database 
+
+Argo run status is logged into a postGIS DB. This is done through an additional docker container (hosted on github container registry `ghcr.io/open-forest-observatory/ofo-argo-utils:latest`) that is included in the argo workflow. The files to make the docker image are in the folder `ofo-argo-utils`. 
+
+<br/>
+
+
+#### Info on the postGIS DB
+There is a JS2 VM called `ofo-postgis` that hosts a postGIS DB for storing metadata of argo workflows. 
+
+<br/>
+
+SSH into ofo-postgis `ssh exouser@149.165.153.127`
```

Comment:
Maybe recommend the webshell instead, or mention they will have to get a password from exosphere? Since it looks like it's not publicly accessible. More broadly, do we want to avoid putting any instance IP addresses online? I don't know anything about what types of attacks are common and/or what information one IP address might provide.